spec/operators/buffer-spec.js

```diff
@@ -2,7 +2,8 @@
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
-describe('Observable.prototype.buffer()', function () {
+/** @test {buffer} */
+describe('Observable.prototype.buffer', function () {
```

Comment:
Usually these changes just add the tag, but in other cases we need to keep it consistent across all files, so I removed the parenthesis